base/doc/ltnews42.tex

```diff
@@ -211,6 +211,28 @@ \subsection{Contexts in typesetting}
 
 \emph{to write if latex-lab-context makes it to the next stage before 2025/11/01}
 
+\subsection{MathML intent attributes}
+Two new commands, \cs{MathMLintent} and \cs{MathMLarg} are added. They are defined
+in the format as no-op so they may be added to command definitions in packages.
+If \pkg{luamml} is enabled to generate MathML, these commands allow \emph{intent} and
+\emph{arg} attributes to be specified.
+
+A  definition such as
+\begin{verbatim}
+\newcommand\abs[1]{%
+\MathMLintent{absolute-value($x)}
```

Comment:
This is quite cool. We tried to come up with a similar macro [before](https://arxiv.org/pdf/2401.16786), but we need people to write \$ instead of $.